Classes/New Issue/IssueTemplates.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  IssueTemplates.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/3/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+struct IssueTemplate {
+    let title:String
+    let template:String
+}
+
+enum IssueTemplateHelper {
+
+    static private func matchesForRegexInText(
+        regex: String!,
+        text: String!) -> [String] {
+
+        do {
+            let regex = try NSRegularExpression(pattern: regex, options: [])
+            let nsString = text as NSString
+
+            let results = regex.matches(
+                in: text,
+                options: [],
+                range: NSMakeRange(0, nsString.length))
+            return results.map { nsString.substring(with: $0.range)}
+
+        } catch let error as NSError {
+            print("invalid regex: \(error.localizedDescription)")
+            return []
+        }
+    }
+
+    static func getNameAndDescriptionFromTemplateFile(file: String) -> (name: String?, about: String?) {
+
+        let names = IssueTemplateHelper.matchesForRegexInText(regex: "(?<=name:).*", text: file)
+        let abouts = IssueTemplateHelper.matchesForRegexInText(regex: "(?<=about:).*", text: file)
+        let name = names.count > 0
+            ? names[0].trimmingCharacters(in: .whitespaces)
+            : nil
+        let about = abouts.count > 0
+            ? abouts[0].trimmingCharacters(in: .whitespaces)
+            : nil
+        return (name, about)
+    }
+
+    static func showIssueAlert(
```

Comment:
It would be cool if a function creates the alert and returns it, so it can be tested.